src/pages/tasks/kanban/Kanban.tsx

```diff
@@ -50,8 +50,9 @@ import { Link as ReactRouterLink } from 'react-router-dom';
 import { Card, Element } from '@invoiceninja/cards';
 import { ProjectSelector } from 'components/projects/ProjectSelector';
 import { Inline } from 'components/Inline';
+import { TaskClock } from './components/TaskClock';
 
-interface Card {
+interface CardItem {
```

Comment:
This change is for security reasons. We have an imported `Card` component and a `Card interface` so just to make a difference I renamed the interface.